### ISSUE TITLE
Help! Something goes wrong when I am running it with vs2012

### ISSUE THREAD

**Issue by mengzixing** [jakesays-old/tesseract-vs2012#2]
libtiff.vcxproj -> C:\Users\kixing\Documents\Visual Studio 2012\Projects\tesseract-vs2012-master\lib\x86\Debug\libtiff.lib
Can't open  tesseract-vs2012-master\bin\x86\debug\liblept168.dll.

**Comment by charlesw** [comment]
I don't believe VS 2012 is supported anymore. The project has been updated to use VS 2013 so try that if possible.

**Comment by charlesw** [comment]
Also what exactly are you trying to accomplish, there may be a way to work around the issue. Such as thinking laterally and all that :)

**Comment by jakesays-old** [comment]
If it's been updated to '13 can you push that back to me? I'll then update it to '15.
